guide/english/react/a-react-environment-using-a-remote-code-repository/index.md

```diff
@@ -5,7 +5,7 @@ title: A React Environment Using a Remote Code Repository
 This is how to create a non production React environment utilizing a remote code repository.
 We will use the cdnjs.cloudflare.com 16.0.0 react, react-dom, and babel-standalone 6.26.0
 to accomplish this.
-babel-polyfill is used for older browsers compatibility.
+babel-polyfill is used for older browsers' compatibility.
```

Comment:
On second thought, I have found that this line sounds very odd when saying out loud
instead of having this you could replace the line `babel-polyfill is used for older browsers' compatibility.` with `babel-polyfill is used for old browser compatibility.`